paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/DefaultConfirmationHandler.kt

```diff
@@ -75,20 +68,17 @@ internal class DefaultConfirmationHandler(
             ),
             BacsConfirmationDefinition(
                 bacsMandateConfirmationLauncherFactory = bacsMandateConfirmationLauncherFactory,
-            )
+            ),
+            googlePayPaymentMethodLauncherFactory?.let {
+                GooglePayConfirmationDefinition(
+                    googlePayPaymentMethodLauncherFactory = it,
+                    userFacingLogger = logger,
+                )
+            }
         )
     )
 
-    private val confirmationMediators = intentConfirmationRegistry.createConfirmationMediators(savedStateHandle)
-
-    private var googlePayPaymentMethodLauncher:
-        ActivityResultLauncher<GooglePayPaymentMethodLauncherContractV2.Args>? = null
-
-    private var currentArguments: ConfirmationHandler.Args?
-        get() = savedStateHandle[ARGUMENTS_KEY]
-        set(value) {
-            savedStateHandle[ARGUMENTS_KEY] = value
-        }
```

Comment:
The parameters are maintained in the `ConfirmationMediator now outside of actual confirmation status.